### DISCUSSION TITLE
Javascript

### DISCUSSION THREAD

**Discussion by amanda4526** [frontendbr/forum#2050] [Perguntas e Respostas]
Faça um fórmulario em Javascript que leia

Preço do litro de combustível em reais;

Consumo médio do veículo em quilômetros por litro.

Distância da cidade onde o usuário se encontra e o seu destino em quilômetros.

Após preencher os campos e clicar em um botão, deve-se chamar uma função em JavaScript que leia estes dados, calcule e apresente quantos reais serão gastos com a viagem no percurso de ida e volta.

Alguém me ajuda?

**Comment by wilsonneto-swe** [comment]
O que já fez? Qual o problema? Qual a dúvida?

**Comment by willstodev** [comment]
Se você quer ajuda para um função ser chamada em um clique, talvez isso esclareça:
```javascript
<button type="button" id="btn-enviar">Enviar</button>

<script>
  function calcConsumoMedio() {};

  document.querySelector('#btn-enviar').onclick = calcConsumoMedio();
  // ou
  document.querySelector('#btn-enviar').addEventListener('click', () => {
    return calcConsumoMedio();
  });
</script>
```

Se o que você quer fazer está dentro de um formulário, é necessário colocar ```type="button"``` na tag button, caso contrário o navegador poderá reiniciar a página.
